CHANGES.rst

```diff
@@ -122,6 +122,11 @@ API Changes
   - Angstrom, erg, G, and barn are no more reported as deprecated FITS units.
     [#5929]
 
+  - Remove compatibility code which forced loading all HDUs on close. The old
+    behavior can be used with `lazy_load_hdus=False`. Because of this change,
```

Comment:
always use double backticks in the changelog, we never do intersphinx linking from here. Fixing this and the next line will get rid of the docs build error on travis, too.